app/views/layouts/_header.html.erb

```diff
@@ -22,6 +22,9 @@
             <% if current_user.present? %>
               <%= link_to "Sign out", signout_path, class: "govuk-header__link" %>
               <%= link_to "(#{current_user['info']['first_name']} #{current_user['info']['last_name']})", "#{Settings.dfe_signin.profile}", class: "govuk-header__link" %>
+              <% if current_user["admin"] %>
+                <%= link_to "Access Requests", access_requests_path, class: "govuk-header__link", data: { qa: "access_requests_link" } %>
```

Comment:
This needs wrapping inside a `<li class="govuk-header__navigation-item">` in order to fix the styling of it.